conf/puppet.conf

```diff
@@ -1,4 +1,4 @@
-[main]
+[master]
     # Where Puppet stores dynamic and growing data.
     vardir=/opt/puppetlabs/agent/cache
```

Comment:
Probably because I just changed the section header and didn't update the settings themselves. That should get updated probably. @shrug did you settle on FOSS master/server paths ever?